### PR TITLE
Add & remove speakers to rooms

### PR DESCRIPTION
**Pull request by ItsEcholot**

### CODE REVIEW

File: frontend/src/pages/EditRoom/index.tsx [hunk 1]
```diff
@@ -18,38 +20,55 @@ const EditRoomPage: FunctionComponent<EditRoomPageProps> = ({
...
+      } else {
+        ack = await createRoom({ name: values.name });
+      }
+      const currentRoomId = currentRoom ? currentRoom.id : ack.createdId;
+
+      if (speakers && currentRoomId) {
+        await Promise.all(speakers.map(speaker => {
+          if (speaker.room.id !== currentRoomId && values.speakers.includes(speaker.id)) {
+            return updateSpeaker({ ...speaker, room: { id: currentRoomId } });
+          } else if (speaker.room.id === currentRoomId) {
```
Comment (cyrilwanner) [inline frontend/src/pages/EditRoom/index.tsx:48]: When I had speakers assigned to a room and simply clicked the save button (e.g. only change the name but nothing related to the speakers), all speakers got removed. I think this is the line that caused that bug.

**Review by cyrilwanner — CHANGES_REQUESTED** [review]
I believe that I have found a bug that deletes all speakers, but otherwise, the code looks good 👍

**Comment by ItsEcholot** [comment]
Thanks for the bug! Never tested just saving the room without changing the speakers 😅 Also re-enabled the disabling of already assigned speakers which I also seemed to forget.

**Review by cyrilwanner — APPROVED** [review]
Thank you, that fixed the bug 👍 
Ah, I thought that was intentionally disabled until it is correctly implemented in the API 😅 But it is of course better to already have it handled correctly here.

**Pull request merged**